tools/go-cli/build.gradle

```diff
@@ -47,7 +49,7 @@ def run(cmd, ignoreError = false) {
     }
 }
 
-task dump_os() << {
```

Comment:
I think keeping this task will be helpful when gathering troubleshooting data from a customer environment.  Just need to run `gradle dump_os` and the pertinent OS values are displayed